server/seeders/20230622202911-addAtBrowser.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.createTable('AtBrowsers', {
+            atId: { type: Sequelize.DataTypes.INTEGER },
+            browserId: { type: Sequelize.DataTypes.INTEGER }
```

Comment:
As it is right, the `atId` and `browserId` foreign key references which links these columns to their respective tables aren't actually created in the database when this is ran.
You should be able to copy the attribute definitions in `models/AtBrowsers.js` and get that link properly created.